---
string[] object_ids  # Ids of objects